app/views/components/page_title.scala.html

```diff
@@ -0,0 +1,24 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(title: String, heading: String)(implicit messages: Messages)
```

Comment:
What is the purpose of this component when we have heading.scala.html?

Reply:
Order of title and hint inside the heading. Prototype changed once again and this part of the page is different than before. Different order and different style. I created a new one because I don't know that we will use old one or not.